controller/uninstall_controller.go

```diff
@@ -804,7 +797,21 @@ func (c *UninstallController) deleteBackupTargets(backupTargets map[string]*long
 	}()
 	for _, bt := range backupTargets {
 		log := getLoggerForBackupTarget(c.logger, bt)
+		if bt.Annotations == nil {
+			bt.Annotations = make(map[string]string)
+		}
 		if bt.DeletionTimestamp == nil {
+			oldURL := bt.Spec.BackupTargetURL
+			_, ok := bt.Annotations[types.GetLonghornLabelKey(types.DeleteBackupTargetFromLonghorn)]
+			bt.Annotations[types.GetLonghornLabelKey(types.DeleteBackupTargetFromLonghorn)] = ""
```

Comment:
Cna you leave a comment for
- What's the purpose of the annotation
- Why is BackupTargetURL set to ""?

Reply:
Added. Thanks.